assets/blocks/single-page-style.scss

```diff
@@ -1,2 +1,6 @@
 @import './learner-courses-block/learner-courses';
 @import './course-results-block/course-results';
+
+.course-list-block .block-editor-block-pattern-setup .block-editor-block-pattern-setup__container .block-editor-block-pattern-setup-list__list-item {
```

Comment:
Just curious if this selector needs to be so specific. Would `.course-list-block .block-editor-block-pattern-setup-list__list-item` work? Also, we should follow the same approach that the other blocks in this file use and put this CSS into an `.scss` file and then import that.

Reply:
So with the new changes, there is no glitching when the patterns are not hidden by default so I removed these css changes altogether. Also when we have modal, the patterns are not wrapped in .course-list-block so this was making query loop block not working :D https://github.com/Automattic/sensei/pull/5466/commits/309b8712712975a4952d1e6ac7e8778a4da6315e#diff-38a475ff59873480ad379d56dd33b083056e15c046a35c9d14ae5c42f8da189bL3